docs/mkdocs.yml

```diff
@@ -0,0 +1,82 @@
+site_name: SKNNR Documentation
```

Comment:
Uppercase or lowercase? Or do we go with the full "scikit-learn-knn-regression"?

Reply:
I have a very slight preference for lowercase, but not a deal-breaker to be uppercase.  For whatever reason, I have an aversion to uppercase words for readability.  

And I agree that we should definitely stick with "sknnr" or "SKNNR" rather than the full repo name.  On the one hand, it might make it a bit more difficult for users to find "scikit-learn-knn-regression" if we always refer to it as "sknnr", but I think that's outweighed by the catchy name.

We might be able to use [pypi-alias](https://github.com/ionelmc/pypi-alias) as a way to publish both "scikit-learn-knn-regression" and "sknnr" to PyPi if I'm understanding the intent correctly.

Reply:
Lowercase sounds good to me!

I hate to rehash an old, settled decision, but I have started wondering if we should reconsider using `sknnr` for the PyPI name. I know we went with the `scikit-learn` prefix to follow the standard for packages that extend `sklearn` functionality, but there's definitely an advantage to the shorter, catchier name, especially since it matches the package. Maybe this is just my bad memory, but I always get confused when I have to install things like Pillow and OpenCV where the PyPI name doesn't match the package. I'm also wondering whether we would continue following the standard with `sknnr-spatial` and call it `scikit-learn-knn-regression-spatial`...

With that said, the connection with `scikit-learn` is definitely important to maintain. If we stick with `scikit-learn-knn-regression`, we should probably still stake out the `sknnr` name on `PyPI` like they do with [`sklearn`](https://pypi.org/project/sklearn/), using an empty package to prevent "impostor" packages.

> sklearn package on PyPI exists to prevent malicious actors from using the sklearn package, since sklearn (the import name) and scikit-learn (the project name) are sometimes used interchangeably.

I doubt `sknnr` will ever be popular enough that someone will use it to distribute malware, but better to be on the safe side!

Reply:
I just reread back through #6 which was a really good discussion (we sound so reasoned ...).  What I came away with from that discussion is that we (I) had initially started with `scikit-learn` as part of the package name, but that we were both really eager to shorter it to something more catchy for dealing with on a day-to-day basis.  I really have no true desire to keep the package name as `scikit-learn-knn-regression`, *other than* to show that it was the inspiration for the name of the package `sknnr`.  

If you feel like moving to `sknnr` as the package name while clearly specifying within the docs the close connection to scikit-learn, I am very much on board.  My only other question is whether to split that out into a separate PR as it will touch a number of files in the package.

Reply:
Thanks for digging back into the archives! As long as you don't have any reservations about switching the published name, that would be my leaning. 

> My only other question is whether to split that out into a separate PR as it will touch a number of files in the package.

I think this is a good idea! I'll open an issue.